lib/chai/utils/checkError.js

```diff
@@ -0,0 +1,132 @@
+/*!
+ * Chai - checkError utility
+ * Copyright(c) 2012-2016 Jake Luer <jake@alogicalparadox.com>
+ * MIT Licensed
+ */
+
+/**
+ * ### .checkError
+ *
+ * Checks that an error conforms to a given set of criteria and/or retrieves information about it.
+ *
+ * @api public
+ */
+
+module.exports =  {
+  compatibleInstance: compatibleInstance,
+  compatibleConstructor: compatibleConstructor,
+  compatibleMessage: compatibleMessage,
+  getMessage: getMessage,
+  getConstructorName: getConstructorName
+};
+
+/**
+ * ### .compatibleInstance(thrown, errorLike)
+ *
+ * Checks if two instances are compatible (strict equal).
+ * Returns true if errorLike is not an instance of Error.
+ *
+ * @name compatibleInstance
+ * @param {Error} thrown error
+ * @param {Error|ErrorConstructor} errorLike object to compare against
+ * @namespace Utils
+ * @api public
+ */
+function compatibleInstance(thrown, errorLike) {
+  if (errorLike instanceof Error) {
+    return thrown === errorLike;
+  } else {
+    return true;
+  }
+}
```

Comment:
That's because of `.throws` behavior. If this method didn't have this check we would need a check back there. Anyway it seems you're right, for the sake of modularity we should keep the check at `throws` and use only a strict comparison here.

I will also update this method's DocStrings, since it should only receive an `Error` instance to compare against and not an `ErrorConstructor`.